### PR TITLE
Autohacker traitor item

### PR DESCRIPTION
**Pull request by BatElite**
## About the PR 
A 2TC traitor item that lets people manipulate airlocks without needing to know how to hack with wires. 

image
(I made the red window smaller just now but you get the idea, also if someone knows how to change the link colours please tell they contrast like shit against that red)

I presume that y'all might have some thoughts on how this should work. Here's the properties as they are right now:

-Hacking an airlock takes 5 seconds the first time, but it keeps track of which airlocks it's done hacking before and those can be accessed instantly the next time
-Styled after the briefcase laptop when folded up for plausible stealthiness
-When you're actually messing with an airlock (or just pop it open IDK) it's pretty obvious though
-Also you have to smack the airlock, which is very obvious and loud. I don't know if that can be suppressed somehow but to me it's a major factor for the instant access thing.
-You can manipulate open/close, bolts, and electrification. Removing electrification requires insuls though, to manage the initial smack.

Basically I'm in a bind where it's either so strong that it could become a goto for anyone, or using it requires so much advance knowledge of the gotchas it's practically useless for the intended audience.

## Why's this needed? 
Discord suggestion, so newbies can do break-ins without needing to understand how hacking works or where they can pilfer a multitool.

## Changelog

```changelog
(u)BatElite
(*)Added the autohacker laptop, a 2TC traitor item that lets you mess with doors.
```

### CODE REVIEW

**Comment by geneva-breaker** [comment]
I'm into it
